README.md

```diff
@@ -13,9 +13,8 @@ define a common infrastructure for the bootloader, system flash layout on
 microcontroller systems, and to provide a secure bootloader that enables
 easy software upgrade.
 
-MCUboot is operating system and hardware independent, and relies on
-hardware porting layers from the operating system it works with.  Currently
-mcuboot works with both the Apache Mynewt, and Zephyr operating systems, but
+MCUboot is operating system and hardware independent and relies on
+hardware porting layers from the operating system it works with.  Currently, mcuboot works with both the Apache Mynewt, and Zephyr operating systems, but
```

Comment:
I am OK with adding a `,` here but please don't rearrange the text (just makes it harder to review) and maintain the line length below 80 chars.

Reply:
Woops sorry, editor auto formatting i'll fix it so it breaks at line breaks at 80